frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -95,7 +95,7 @@ pods:
             kafka_server_log_files=${kafka_dir}/logs/server.log*
```

Comment:
Just a thought. Would it make sense to explicitly set `kafka_dir=$MESOS_SANDBOX/{{KAFKA_VERSION_PATH}}` instead of relying on the `ls` magic?

Reply:
(This would be one less change required for updates to confluent)

Reply:
That would certainly work, yeah.